### PR TITLE
solution

### PR DESCRIPTION
**Pull request by RomSmile**
DEMO LINK: https://RomSmile.github.io/react_article/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
+  />
```
Comment (zadorojniiarsen) [inline src/App.js:17]: You can reduce amount of code

```suggestion
  <Article {...article} />
```

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { string } from 'prop-types';
+import React from 'react';
+
+export const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
```
Comment (zadorojniiarsen) [inline src/components/Article.jsx:9]: You can use semantic tag <article>

Comment (varseniuk) [reply src/components/Article.jsx:9]: Arsen is right. Use `<article>` tag instead of a fragment

**Review by varseniuk — CHANGES_REQUESTED** [review]
All is good, but let's use some semantics

**Review by varseniuk — APPROVED** [review]

**Pull request closed without merging**